### PR TITLE
Python Implementation

### PR DESCRIPTION
**Pull request by nour-massri**

### CODE REVIEW

**Review by nwatx — APPROVED** [review]

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

File: solutions/ac-IndependentSet.mdx [hunk 1]
```diff
@@ -151,5 +151,48 @@ public class Main {
...
+```py
+# to solve Runtime Error problem with recursion in python
+from sys import setrecursionlimit
+setrecursionlimit(10**9)
+
+n = int(input())
+mod = 10**9 + 7
+
+# representing the tree in an adjacency list
+adj = [[] for i in range(n)]
```
[the remark this answers is not in the document]
Comment (nwatx) [reply solutions/ac-IndependentSet.mdx:165]: image

[the remark this answers is not in the document]
Comment (chezbgone) [reply solutions/ac-IndependentSet.mdx:165]: If you use `*` for arrays, the elements in the array all have the same reference, which is not what we want.

**Comment by Aadit-Ambadkar** [comment]
Removed my incorrect revisions, sorry about that. Silly mistake on my part.

**Review by Aadit-Ambadkar — APPROVED** [review]
Looks Good

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**